src/buffer_info.cpp

```diff
@@ -162,7 +162,10 @@ MultiBufferWorkspaceTraits::MultiBufferWorkspaceTraits(std::initializer_list<siz
     }
 }
 
-size_t MultiBufferWorkspaceTraits::GetSize() const { return v_offset.back(); }
+size_t MultiBufferWorkspaceTraits::GetSize() const
+{
+    return v_offset[v_offset.size() - 2] + v_size.back();
```

Comment:
[Recommendation] Maybe you'll find this nicer
```suggestion
    return (&v_offset.back())[-1] + v_size.back();
```